mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -402,13 +345,13 @@ class VizReads(protected val args: VizReadsArgs) extends BDGSparkCommand[VizRead
         args.partitionCount
 
     // initialize all datasets
-    initReference()
-    initAlignments()
-    initVariants()
-    initFeatures()
```

Comment:
Random semantic question- is it good to have parenthesis or not? IntelliJ warns to use parenthesis, not sure if that's correct though.